proto/connectrpc/conformance/v1/client_compat.proto

```diff
@@ -114,41 +136,53 @@ message ClientCompatRequest {
 
 // The outcome of one ClientCompatRequest.
 message ClientCompatResponse {
+  // The test name that this response applies to.
   string test_name = 1;
+  // These fields determine the outcome of the request. Note that an `error`
+  // should only be reported for unexpected internal errors. RPC errors, such
+  // as Connect error codes, should be reported as part of the `response`.
```

Comment:
Another way to say this, which might help clarify, is that any unexpected errors that prevent the client from issuing the RPC and following the instructions implied by the request can be reported as an `error`. These would be errors creating an RPC client from the request parameters or unsupported/illegal values in the request (e.g. a unary request that defines zero or multiple request messages). But once the RPC is issued, any resulting error should instead be encoded in `response`.